tests/lang/s05_pragmas/s01_interop/t02_cxx_header.hpp

```diff
@@ -10,11 +10,39 @@ struct __cxx_internal{
   CxxForward* forward;
 };
 
-template <typename T1, typename T2, typename T3 = int>
+struct CxxComplete {
```

Comment:
For a future improvement, we should split out the backend code or platform specific pragmas into their own thing -- it's not very basic or simple and they should come fairly late.

Moreover, as these are the ones that are most likely to be poorly thought out in the first place we want the fewest things depending upon them, even in testing.

This just needs a good place to be noted down so we don't ossify this decision.